app/config.js

```diff
@@ -41,4 +41,4 @@ window.OPENSHIFT_VERSION = {
   kubernetes: "dev-mode"
 };
 
-window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:9000/dev-console";
+window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:" + (window.DEV_SERVER_PORT || '9001');
```

Comment:
This is in the showcase app only. It breaks in there unless you happen to have the console running.